library/js/xl/jquery-datetimepicker-2-5-4.js.php

```diff
@@ -41,6 +41,8 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
```

Comment:
will need to ensure the new library/js/xl/jquery-datetimepicker-2-5-4-translated.js has same logic for the fixed time stuff